test.sh

```diff
@@ -3,10 +3,22 @@
 cd ./BEIMA.Backend.Test
 # Run backend unit tests
 dotnet test ./BEIMA.Backend.Test.csproj || { echo "Unit tests failed"; exit 1; }
-# Change back to root directory
+# Navigate to backend project
+cd ../
+cd ./BEIMA.Backend
+# Start the backend API locally
+func start BEIMA.Backend.csproj --csharp &
```

Comment:
The '&' at the end allows us to start the API in the background without having to wait for it to exit to continue on to the rest of the code.